lib/rollup/data/trends_data.ex

```diff
@@ -1,34 +1,52 @@
 defmodule Castle.Rollup.Data.Trends do
-  import Castle.Rollup.Jobs.Trends
+  alias Castle.Redis.Interval.Getter, as: Getter
+
+  # TODO: this is a bit inside-baseball
+  @podcast_downloads_key "downloads.podcasts.DAY"
+  @episode_downloads_key "downloads.episodes.DAY"
+  @daily BigQuery.TimestampRollups.Daily
 
   def podcast(id) do
-    {result, _meta} = get()
-    result
-    |> Enum.filter(&(&1.feeder_podcast == id))
-    |> sum_fields()
+    {last7, this7, yesterday, today, tomorrow} = get_dates()
+    hits = get_from_interval(@podcast_downloads_key, id, last7, tomorrow)
```

Comment:
And this _just_ gets 14 days from the `downloads.podcasts.DAY.2018-02-16T00:00:00Z` cache keys, and sums them up.